dynatrace-oneagent-operator/README.md

```diff
@@ -39,20 +39,31 @@ Generate an API and a PaaS token in your Dynatrace environment.
 
 https://www.dynatrace.com/support/help/reference/dynatrace-concepts/why-do-i-need-an-environment-id/#create-user-generated-access-tokens
 
-To install the Dynatrace OneAgent Operator replace the APIUrl, the API token and the PaaS token in command and execute it
+### Chart installation
+
+To install the Dynatrace OneAgent Operator first create the dynatrace namespace, apply the latest CRD from [the latest release](https://github.com/Dynatrace/dynatrace-oneagent-operator/releases/latest) and replace the APIUrl, the API token and the PaaS token in command and execute it
 
 #### Kubernetes
 ```
 $ kubectl create namespace dynatrace
+$ kubectl apply -f https://github.com/Dynatrace/dynatrace-oneagent-operator/releases/latest/download/dynatrace.com_dynakubes.yaml
 $ helm install dynatrace-oneagent-operator dynatrace/dynatrace-oneagent-operator -n dynatrace --set platform="kubernetes",oneagent.apiUrl="https://ENVIRONMENTID.live.dynatrace.com/api",secret.apiToken="DYNATRACE_API_TOKEN",secret.paasToken="PLATFORM_AS_A_SERVICE_TOKEN"
 ```
 
 #### OpenShift
 ```
 $ oc adm new-project --node-selector="" dynatrace
+$ oc apply -f https://github.com/Dynatrace/dynatrace-oneagent-operator/releases/latest/download/dynatrace.com_dynakubes.yaml
 $ helm install dynatrace-oneagent-operator dynatrace/dynatrace-oneagent-operator -n dynatrace --set platform="openshift",oneagent.apiUrl="https://ENVIRONMENTID.live.dynatrace.com/api",secret.apiToken="DYNATRACE_API_TOKEN",secret.paasToken="PLATFORM_AS_A_SERVICE_TOKEN"
 ```
 
+##### OpenShift 3.11
+```
+$ oc adm new-project --node-selector="" dynatrace
+$ oc apply -f https://github.com/Dynatrace/dynatrace-oneagent-operator/releases/latest/download/dynatrace.com_dynakubes-crd-v1beta1.yaml
```

Comment:
```suggestion
$ oc apply -f https://github.com/Dynatrace/dynatrace-oneagent-operator/releases/latest/download/dynatrace.com_dynakubes-v1beta1.yaml
```

Reply:
super nitpick: `dynatrace.com_dynakubes-v1beta1.yaml` is rather ugly, it has a `.` a `_` and `-`  in the name, I would just choose one, 
so `dynatrace.com.dynakubes.v1beta1.yaml`, `dynatrace-com-dynakubes-v1beta1.yaml`, `dynatrace_com_dynakubes-v1beta1.yaml`